tcms/rpc/api/environment.py

```diff
@@ -110,3 +111,27 @@ def filter(query=None):  # pylint: disable=redefined-builtin
         .order_by("name", "description")
         .distinct()
     )
+
+
+@permissions_required("testruns.add_environment")
+@rpc_method(name="Environment.add_environment")
+def add_environment(name, description=None):
```

Comment:
See signatures for the rest of the `create()` methods in the API.  We'd accept a `values` dictionary to allow for easier expansion in the future.

Reply:
Done in https://github.com/kiwitcms/Kiwi/pull/3082/commits/a1e6436b01353a29ed8dbe31e146b6eb308bf693